app/uk/gov/hmrc/thirdpartypaymentsexternalapi/models/thirdparty/ThirdPartyPayRequest.scala

```diff
@@ -20,19 +20,31 @@ import play.api.libs.json.{Json, OFormat}
 import uk.gov.hmrc.thirdpartypaymentsexternalapi.models.payapi.{SpjRequest3psCorporationTax, SpjRequest3psEmployersPayAsYouEarn, SpjRequest3psSa, SpjRequest3psVat}
 import uk.gov.hmrc.thirdpartypaymentsexternalapi.models.{ClientJourneyId, TaxRegime}
 
-final case class ThirdPartyPayRequest(taxRegime: TaxRegime, reference: String, amountInPence: Int, backURL: String) {
+import java.time.LocalDate
+
+final case class ThirdPartyPayRequest(
+    taxRegime:     TaxRegime,
+    reference:     String,
+    amountInPence: Int,
+    friendlyName:  Option[String],
+    backURL:       String,
+    dueDate:       Option[LocalDate]
```

Comment:
We don't want dueDate brought back in, it was removed in [OPS-13263](https://jira.tools.tax.service.gov.uk/browse/OPS-13263)